integration_tests/commands/websocket/main_test.go

```diff
@@ -37,8 +38,8 @@ func TestMain(m *testing.M) {
 	// abort
 	conn := executor.ConnectToServer()
 	executor.FireCommand(conn, "abort")
-	executor.DisconnectServer(conn)
 
+	cancel()
```

Comment:
Shall we just `defer cancel()` after L27?

Reply:
@lucifercr07 - fixed.